### PR TITLE
Translation and trn btn

### PR DESCRIPTION
**Pull request by alionachykal**
made translation btn and  add use i18n

### CODE REVIEW

File: src/components/RegisterForm/index.jsx [hunk 1]
```diff
@@ -1,79 +1,115 @@
 import { validationSchema } from 'schemas/registerFormValidation';
+import { useTranslation } from 'react-i18next';
 
 import { Input } from 'core/kit/Input';
 import { Button } from 'core/kit/Button';
 
 import { Formik } from 'formik';
 
+const LangaguesBar = () => {
```
Comment (kenobiwins) [inline src/components/RegisterForm/index.jsx:9]: Why ???

File: src/core/i18n/i18n.js [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
-import Backend from "i18next-http-backend";
+import i18next from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import Backend from 'i18next-http-backend';
 // to do translaion.json
-const FRONTEND_BASE_URL = "http://localhost:3000";
+const FRONTEND_BASE_URL = 'http://localhost:3000';
+
+const resources = {
```
Comment (kenobiwins) [inline src/core/i18n/i18n.js:8]: why ??? 
pubic/locales for this

Comment (alionachykal) [reply src/core/i18n/i18n.js]: fixed

File: src/core/i18n/i18n.js [hunk 3]
```diff
@@ -2,47 +2,21 @@ import i18next from 'i18next';
...
+// const FRONTEND_BASE_URL = 'http://localhost:3000';
 
 i18next
     .use(initReactI18next)
     .use(LanguageDetector)
     .use(Backend)
     .init({
         backend: {
-            loadPath: `${FRONTEND_BASE_URL}/goosetrack-team-project-front/locales/{{lng}}/translation.json`,
+            loadPath: `/goosetrack-team-project-front/locales/{{lng}}/translation.json`,
```
Comment (kenobiwins) [inline src/core/i18n/i18n.js:15]: base url?

Comment (alionachykal) [reply src/core/i18n/i18n.js]: my fault , fixed

**Review by SerjKenri — APPROVED** [review]

**Pull request merged**